app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/cohort/CohortPixelInterceptor.kt

```diff
@@ -68,10 +70,12 @@ class CohortPixelInterceptor @Inject constructor(
     }
 
     companion object {
-        private const val COHORT_PARAM = "atp_cohort"
+        @VisibleForTesting
+        const val COHORT_PARAM = "atp_cohort"
         private const val PIXEL_PREFIX = "m_atp_"
         private val EXCEPTIONS = listOf(
             "m_atp_ev_enabled_onboarding_",
+            PixelPrefixes.ATP_APP_CPU_MONITOR_REPORT_PREFIX,
```

Comment:
Hardcode the pixel name, that also guards against unintended pixel name changes
```suggestion
            "m_atp_ev_cpu_usage_",
```